### PR TITLE
replace $VSC_SCRATCH_PHANPY with $VSC_SCRATCH_ARCANINE

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: intro-HPC/ch_running_jobs_with_input_output_data.tex [hunk 1]
```diff
@@ -171,7 +171,7 @@ \subsection{Pre-defined user directories}
                          shared across all clusters of the VSC in the future. See \S\ref{subsec:scratch-directory}. \\ \hline
 \ifgent
 \$VSC\_SCRATCH\_CLUSTER & The scratch filesystem closest to this cluster.\\ \hline
-\$VSC\_SCRATCH\_PHANPY  & A separate (smaller) shared scratch filesystem, powered by SSDs. This scratch filesystem is intended for very I/O-intensive workloads.\\ \hline
+\$VSC\_SCRATCH\_ARCANINE  & A separate (smaller) shared scratch filesystem, powered by SSDs. This scratch filesystem is intended for very I/O-intensive workloads. Note that this filesystem is not mounted on the cluster victini.\\ \hline
```
Comment (boegel) [inline intro-HPC/ch_running_jobs_with_input_output_data.tex:174]: `mounted` -> `available` (or `accessible`)

**Review by boegel — CHANGES_REQUESTED** [review]

File: intro-HPC/ch_running_jobs_with_input_output_data.tex (hunk 1 above)
Comment (hajgato) [reply intro-HPC/ch_running_jobs_with_input_output_data.tex]: @boegel fixed

**Review by boegel — APPROVED** [review]
lgtm

**Comment by hajgato** [comment]
@boegel do you know why is it fails?

**Comment by hajgato** [comment]
I mean travis.

**Comment by boegel** [comment]
No, not really, no clear error...

@wpoely86 Any ideas?

**Comment by wpoely86** [comment]
No clue. It works locally for me. Judging by the error, it might be a travis issue. Did you try restart the tests?

**Comment by boegel** [comment]
I restarted them once, trying again now...

**Comment by boegel** [comment]
Keeps failing... :-/

One option we could try is switching to GitHub Actions for the CI, but that's not entirely trivial I think...

**Comment by wpoely86** [comment]
@hajgato can you merge/rebase on master? It should now use github actions instead of travis.

**Pull request merged**